cost-analyzer/values.yaml

```diff
@@ -489,6 +489,7 @@ serviceAccount:
 #  serviceKeySecretName: "" # Use an existing AWS or Azure secret with format as in aws-service-key-secret.yaml or azure-service-key-secret.yaml. Leave blank if using createServiceKeySecret 
 #  createServiceKeySecret: true # Creates a secret representing your cloud service key based on data in values.yaml. If you are storing unencrypted values, add a secret manually
 #  sharedNamespaces: ""
-#  productKey: # apply business or enterprise product license
+  productKey: # apply business or enterprise product license. Supply as plaintext (this is never used to access your data) or as an existing secret
 #    key: ""
 #    enabled: false
+    secretname: productkeysecret # create a secret of format 
```

Comment:
Is this an existing secret? If so, should we just call it `existingProductKeySecret`?

Also, I'm assuming we want this empty of commented out?

Reply:
"existing" is a bit inconsistent with how we do this for serviceKeySecretName. Definitely needs to be commented by default though.

Reply:
I personally think how this is currently done is not intuitive.